distribution/docs/src/main/resources/content/_integrating/_endpoints/csw-endpoint.adoc

```diff
@@ -1159,121 +362,27 @@ It is possible to receive a response to a `GetRecords` query that conforms to th
-                        <gco:CharacterString>en</gco:CharacterString>
-                    </language>
-                    <extent>
-                        <EX_Extent>
-                            <geographicElement>
-                                <EX_GeographicBoundingBox>
-                                    <westBoundLongitude>
-                                        <gco:Decimal>32.975277</gco:Decimal>
-                                    </westBoundLongitude>
-                                    <eastBoundLongitude>
-                                        <gco:Decimal>32.996944</gco:Decimal>
-                                    </eastBoundLongitude>
-                                    <southBoundLatitude>
-                                        <gco:Decimal>32.305</gco:Decimal>
-                                    </southBoundLatitude>
-                                    <northBoundLatitude>
-                                        <gco:Decimal>32.323333</gco:Decimal>
-                                    </northBoundLatitude>
-                                </EX_GeographicBoundingBox>
-                            </geographicElement>
-                        </EX_Extent>
-                    </extent>
-                </MD_DataIdentification>
-            </identificationInfo>
-            <distributionInfo>
-                <MD_Distribution>
-                    <distributor>
-                        <MD_Distributor>
-                            <distributorContact/>
-                            <distributorTransferOptions>
-                                <MD_DigitalTransferOptions>
-                                    <onLine>
-                                        <CI_OnlineResource>
-                                            <linkage>
-                                                <URL>http://example.com</URL>
-                                            </linkage>
-                                        </CI_OnlineResource>
-                                    </onLine>
-                                </MD_DigitalTransferOptions>
-                            </distributorTransferOptions>
-                        </MD_Distributor>
-                    </distributor>
-                </MD_Distribution>
-            </distributionInfo>
-        </MD_Metadata>
-    </csw:SearchResults>
-</csw:GetRecordsResponse>
-----
 
-.`GetRecords` XML Request using UTM Coordinates
+.Querying by UTM Coordinates
 UTM coordinates can be used when making a CSW GetRecords request using an `ogc:Filter`.
-UTM coordinates should use EPSG:326XX as the srsName where XX is the zone within the northern hemisphere.
-UTM coordinates should use EPSG:327XX as the srsName where XX is the zone within the southern hemisphere.
+UTM coordinates should use EPSG:326XX as the `srsName` where `XX` is the zone within the northern hemisphere.
+UTM coordinates should use EPSG:327XX as the `srsName` where `XX` is the zone within the southern hemisphere.
```

Comment:
✏️ Should EPSG:326XX & EPSG:327XX be surrounded by \`\` ? It may flow better, especially since the `XX`s are surrounded.

Reply:
that's a good idea